ietf/templates/includes/header_iab.html

```diff
@@ -0,0 +1,133 @@
+{% load static wagtailcore_tags %}
+
+<header class="header" >
+    <a href="#content" class="visually-hidden visually-hidden-focusable">Skip to main content</a>
+    <nav
+        class="navbar navbar-expand-lg navbar-dark bg-brand-blue-iab text-light"
+        aria-label="Main"
+    >
+        <div class="container">
+            <!-- Brand Image -->
+            <div class="navbar-brand">
+                <a class="text-light" href="{{ HOME.url }}">
+                    <img
+                        class="header__logo"
+                        src="{% static 'img/iab-logo.svg' %}"
+                        alt="IETF logo"
+                    />
+                </a>
+            </div>
+            <!-- Brand Text -->
+            <div class="logo-text-iab">
+                Internet Architecture Board
+            </div>
+
+            <!-- Mobile search -->
+            <button
+                class="btn btn-link nav-link text-light ms-auto d-lg-none"
+                type="button"
+                data-bs-toggle="collapse"
+                data-bs-target="#navbarSearch"
+                aria-controls="navbarSearch"
+                aria-expanded="false"
```

Comment:
I'm assuming the aria-expanded value gets updated on click? Is that handled by bootstrap JS?